mlflow/genai/judges/databricks.py

```diff
@@ -339,3 +340,58 @@ def meets_guidelines(
             assessment_name=name,
         )
     )
+
+
+@experimental
+@requires_databricks_agents
+def custom_prompt_judge(
+    *,
+    name: str,
+    prompt_template: str,
+    numeric_values: Union[dict[str, Union[int, float]], None] = None,
+) -> Callable[..., Assessment]:
```

Comment:
```suggestion
) -> Callable[..., Feedback]:
```

Judge will never return `Expectation`.